doc/releases/release-notes-3.3.rst

```diff
@@ -1064,30 +1064,827 @@ USB
 Devicetree
 **********
 
-* Bindings
+API
+===
 
-  * New:
+New general-purpose macros:
 
-    * :dtcompatible:`zephyr,flash-disk`
+- :c:macro:`DT_FOREACH_PROP_ELEM_SEP_VARGS`
```

Comment:
We have been recommending using `*` for bullet points for the sake of consistency, but I can batch-fix these later.

Reply:
> We have been recommending using `*` for bullet points

FYI, I have updated the script accordingly; thanks for the heads-up